src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/PseRespondentViewServiceTest.java

```diff
@@ -77,6 +83,146 @@ void populateSelectDropdownView_checkSendNotificationNotify_returnList() {
+                    .sendNotificationSubject(List.of("Hearing", "Case management orders / requests"))
+                    .sendNotificationSelectHearing(DynamicFixedListType.of(
+                        DynamicValueType.create("3", "3: Hearing - Leeds - 14 Aug 2022")))
+                    .sendNotificationCaseManagement("Case management order")
+                    .sendNotificationResponseTribunal("Yes - view document for details")
+                    .sendNotificationSelectParties(BOTH_PARTIES)
+                    .sendNotificationWhoCaseOrder("Legal Officer")
+                    .sendNotificationFullName("Mr Lee Gal Officer")
+                    .sendNotificationAdditionalInfo("Additional Info")
+                    .sendNotificationNotify(BOTH_PARTIES)
+                    .respondCollection(List.of(pseResponseTypeItem))
+                    .build())
+                .build()
+        ));
+
+        caseData.setPseRespondentSelectJudgmentOrderNotification(
+            DynamicFixedListType.of(DynamicValueType.create("1",
+                "1 View notice of hearing")));
+
+        String expected = "### Hearing, case management order or request \r\n "
+            + "| | |\r\n"
+            + "|--|--|\r\n"
+            + "|Notification | View notice of hearing|\r\n"
+            + "|Hearing | 3: Hearing - Leeds - 14 Aug 2022|\r\n"
+            + "|Date sent | 5 Aug 2022|\r\n"
+            + "|Sent by | Tribunal|\r\n"
+            + "|Case management order or request? | Case management order|\r\n"
+            + "|Response due | Yes - view document for details|\r\n"
+            + "|Party or parties to respond | Both parties|\r\n"
+            + "|Additional information | Additional Info|\r\n"
+            + "|Description | Notice of Hearing and Submit Hearing Agenda document|\r\n"
+            + "|Document | <a href=\"/documents/5fac5af5-b8ac-458c-a329-31cce78da5c2/binary\" target=\"_blank\">Letter 4.8 - Hearing notice - hearing agenda.pdf</a>|\r\n"
+            + "|Case management order made by | Legal Officer|\r\n"
+            + "|Name | Mr Lee Gal Officer|\r\n"
+            + "|Sent to | Both parties|\r\n"
+            + "\r\n"
+            + "|Response 1 | |\r\n"
+            + "|--|--|\r\n"
+            + "|Response from | Claimant|\r\n"
+            + "|Response date | 10 Aug 2022|\r\n"
+            + "|What's your response to the tribunal? | Response text entered|\r\n"
+            + "|Supporting material | <a href=\"/documents/ca35bccd-f507-4243-9133-f6081fb0fe5e/binary\" target=\"_blank\">My claimant hearing agenda.pdf</a>\r\n|\r\n"
+            + "|Do you want to copy this correspondence to the other party to satisfy the Rules of Procedure? | "
+            + "Yes|\r\n"
+            + "\r\n";
+
+        assertThat(pseRespondentViewService.initialOrdReqDetailsTableMarkUp(caseData),
+            is(expected));
+    }
+
+    private DocumentTypeItem createDocumentTypeItem(String fileName, String uuid) {
```

Comment:
Can we reduce them to one method and just assign null when description is not needed? it just seems redundant to have two methods like this.